src/main/java/frc/robot/RobotContainer.java

```diff
@@ -616,4 +623,12 @@ public void displaySimFieldToAdvantageScope() {
     if (Constants.currentMode != Constants.Mode.SIM) return;
     mapleSimArenaSubsystem.updateRobotCoralPose(elevator.getElevatorHeight());
   }
+
+  public Command automaticScore() {
+    if (coralScorer.scorerAlignedTrigger().getAsBoolean()) {
+      return coralScorer.scoreCommand();
+    } else {
+      return Commands.none();
+    }
+  }
```

Comment:
The Timer logic is unnecessary. Just do the same thing we are doing to set the controller rumble, but then just add a .denounce of x amount of time. It will do the same thing in a much cleaner fashion.